### PR TITLE
DDF-2617: Configuring solr to log to separate log file to reduce noise in main log.

### PR DESCRIPTION
**Pull request by brjeter**
#### What does this PR do?
Configures all logs from `org.apache.solr` to go to a separate log file instead of in the main log. This is to reduce noise in the main log, but still have solr logs for debugging.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@Lambeaux @figliold @ricklarsen 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Solr](https://github.com/orgs/codice/teams/solr)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@shaundmorris 
#### How should this be tested? (List steps with links to updated documentation)
- Build and run (including stepping through setup).
- Restart DDF. This will produce some solr errors when trying to create the Solr cores (see below for example). `log:tail` and verify there are no such errors in the console. Check `{karaf.home}/data/log/solr.log` and verify the solr logs are there including the errors.

```ERROR | qtp255340046-91  | org.apache.solr.common.SolrException      148 | lr-server-standalone | org.apache.solr.common.SolrException: Error CREATEing SolrCore 'metacard_cache': Could not create a new core in /opt/ddf/data/solr/metacard_cacheas another core is already defined there```
#### What are the relevant tickets?
[DDF-2617](https://codice.atlassian.net/browse/DDF-2617)
#### Checklist:
- [X] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/etc/log4j2.config.xml [hunk 1]
```diff
@@ -75,6 +75,16 @@
                 <AppenderRef ref="securityBackup"/>
             </Failovers>
         </Failover>
+
+        <RollingFile name="solr" append="true"
+                     fileName="${sys:karaf.data}/log/solr.log"
+                     filePattern="${sys:karaf.data}/log/ingest_error.log-%d{yyyy-MM-dd-HH}-%i.log.gz">
```
Comment (pklinef) [inline distribution/ddf-common/src/main/resources/etc/log4j2.config.xml]: ingest_error.log should be solr.log on this line.  Looks like a copy and paste mistake.

File: distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg [hunk 2]
```diff
@@ -92,7 +105,6 @@ log4j.appender.sift.appender.append=true
 # CXF and Solr logging is verbose.  Default setting to WARN.  This can be changed in the karaf console.
 log4j.logger.org.apache.cxf = WARN
 log4j.logger.org.apache.lucene = WARN
```
Comment (pklinef) [inline distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg]: I would suggest adding Lucene to the Solr log file.

File: distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc [hunk 3]
```diff
@@ -314,6 +314,9 @@ During ${branding} installation, the major directories and files shown in the ta
 |`data/log/security.log`
 |Log file that records user interactions with the system for auditing purposes.
 
+|`data/log/solr.log`
+|Log file for any info coming from solr.
```
Comment (pklinef) [inline distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc]: I would reword to "from Standalone Solr Server".  I think that is how we usually refer to it in documentation.  Does that sound right @ricklarsen?

File: distribution/docs/src/main/resources/_contents/installing-contents.adoc [hunk 4]
```diff
@@ -316,6 +316,9 @@ During ${branding} installation, the major directories and files shown in the ta
 |`data/log/security.log`
 |Log file that records user interactions with the system for auditing purposes.
 
+|`data/log/solr.log`
+|Log file for any info coming from solr.
```
Comment (pklinef) [inline distribution/docs/src/main/resources/_contents/installing-contents.adoc]: Same comment as above.

**Comment by Lambeaux** [comment]
Hero has passed 👍
